app/controllers/netflix_shows_controller.rb

```diff
@@ -2,4 +2,8 @@ class NetflixShowsController < ApplicationController
   def index
     @netflix_shows = NetflixShow.all
   end
+
+  def show
+    @netflix_show = NetflixShow.find(params[:id])
+  end
 end
```

Comment:
This controller action isnt needed since the page is being rendered in react and we are getting our data through an Api::V1 endpoint.